examples/src/main/java/com/teamdev/jxbrowser/examples/DomQuerySelector.java

```diff
@@ -45,6 +47,12 @@ public static void main(String[] args) {
             BrowserView view = BrowserView.newInstance(browser);
 
             JFrame frame = new JFrame("DOM Query Selector");
+            frame.addWindowListener(new WindowAdapter() {
+                @Override
+                public void windowClosing(WindowEvent e) {
+                    engine.close();
+                }
+            });
```

Comment:
@vladimir-ikryanov, please look into the number of repetitions of the same code. Why don't we have one class which does this for all the examples?

Reply:
I suppose it's because we want to make each example standalone, so that developers can copy and paste it easily.